package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nypl/design-system-react-components",
-  "version": "3.1.0",
+  "version": "3.1.1-modal-rc",
```

Comment:
Did you make a new `rc-2` release to test the latest changes in RC?

Reply:
I haven't yet but i will!